tutorials/migration-guide.html

```diff
@@ -1,6 +1,221 @@
 <div class="static-content">
+  <div class="index-list">
+    <ul>
+      <li>
+        <a href="#migrating-from-8-to-9">Migrating from 8 to 9</a>
+        <ul>
+          <li><a href="#9.0.0-formulas-plugin">Formulas plugin</a></li>
+          <li><a href="#9.0.0-autofill-hooks">Autofill hooks</a></li>
+          <li><a href="#9.0.0-removed-plugins">Removed plugins</a></li>
```

Comment:
Can we move plugins sections up in the hierarchy?

```suggestion
          <li><a href="#9.0.0-removed-plugins">Removed ObserceChanges plugin</a></li>
          <li><a href="#9.0.0-removed-plugins">Removed HeaderTooltip plugin</a></li>
```
Is easier to skim through the breaking changes and does not trigger a panic attack "which ones???!!" ;)